tests/testthat/test-optimise.R

```diff
@@ -0,0 +1,43 @@
+library(ROI.plugin.glpk)
```

Comment:
Not sure of the correct way to do this - it's only required for its side-effects (i.e. loading the plugin), and apparently needs to be done in the context of where the code is invoked, not defined (i.e. it isn't sufficient to put this line in `optimise.R`.